Corona-Warn-App/src/main/res/layout/fragment_information.xml

```diff
@@ -31,28 +31,23 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/information_header">
 
-            <androidx.constraintlayout.widget.ConstraintLayout
+            <LinearLayout
```

Comment:
I don't think this will be necessary here – it's not "intentionally" a `LinearLayout` in the sense that it is not somehow better than other options besides personal preference, and besides that it makes no sense to stack `ConstraintLayout`s